examples/react-final-form/js/validate-form/index.js

```diff
@@ -17,13 +17,7 @@ const App = () => (
     <h2>Password / Confirm Validation</h2>
     <Form
       onSubmit={onSubmit}
-      validate={values =>
-        formValidation
-          .validateForm(values)
-          .then(validationResult =>
-            validationResult ? validationResult.fieldErrors : null
-          )
-      }
+      validate={values => formValidation.validateForm(values)}
```

Comment:
@nasdan this is strange, this code works, but if I use

```javascript
validate={formValidation.validateForm}
```
I've got a bad crash, any idea why this can happen? Just thinking loud the import is donde before component instantiation so formValidation is defined and hence validateForm

Reply:
I think is because we are invoking an outer class method, could it be that?